build/Nuget/Microsoft.VisualStudio.ProjectSystem.nuget.proj

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I am modifying the name to `Microsoft.VisualStudio.ProjectSystem.VSInsertionNuget.proj`